steering-files/src/main/resources/org/hps/steering/recon/PhysicsRun2019MCRecon_KF_NoSpacing.lcsim

```diff
@@ -0,0 +1,208 @@
+            <useTruthTime>false</useTruthTime>    
+            <!--time of flight corrections-->              
+            <subtractTOF>true</subtractTOF>     
+            <!--set this false for MC, true for data-->              
+            <subtractTriggerTime>false</subtractTriggerTime>           
+            <!--per-strip timing correction from database...this should be on i f <useTimingConditions> is turned on in readout  -->  
+            <correctChanT0>true</correctChanT0>          
+            <isMC>true</isMC>
+            <debug>false</debug>
+        </driver>
+        <!-- 
+             Use the fitted raw tracker hits and create 1D clusters using a 
+             nearest neighbor algorithm. 
+        --> 
+        <driver name="TrackerHitDriver" type="org.hps.recon.tracking.DataTrackerHitDriver">
+	    <neighborDeltaT>24.0</neighborDeltaT>
+            <neighborDeltaTSigma>3.0</neighborDeltaTSigma>
+            <saveMonsterEvents>false</saveMonsterEvents>
+            <thresholdMonsterEvents>400</thresholdMonsterEvents>
+            <clusterSeedThreshold>4.0</clusterSeedThreshold>
+            <doTimeError>1.0</doTimeError>
+            <clusterNeighborThreshold>3.0</clusterNeighborThreshold>
+            <clusterThreshold>3.0</clusterThreshold>
+            <doDeadFix>true</doDeadFix>
+            <doVSplit>true</doVSplit>
+	    <debug>false</debug>
+        </driver>
+    
+        <driver name="ReconParticleDriver_Kalman" type="org.hps.recon.particle.HpsReconParticleDriver" > 
+            <ecalClusterCollectionName>EcalClustersCorr</ecalClusterCollectionName>
+            <trackCollectionNames>KalmanFullTracks</trackCollectionNames>
+            <matcherTrackCollectionName>KalmanFullTracks</matcherTrackCollectionName>
+            <trackClusterMatcherAlgo>TrackClusterMatcherMinDistance</trackClusterMatcherAlgo>
+            <unconstrainedV0CandidatesColName>UnconstrainedV0Candidates_KF</unconstrainedV0CandidatesColName>
+            <unconstrainedV0VerticesColName>UnconstrainedV0Vertices_KF</unconstrainedV0VerticesColName>
+            <beamConV0CandidatesColName>BeamspotConstrainedV0Candidates_KF</beamConV0CandidatesColName>
+            <beamConV0VerticesColName>BeamspotConstrainedV0Vertices_KF</beamConV0VerticesColName>
+            <targetConV0CandidatesColName>TargetConstrainedV0Candidates_KF</targetConV0CandidatesColName>
+            <targetConV0VerticesColName>TargetConstrainedV0Vertices_KF</targetConV0VerticesColName>
+            <finalStateParticlesColName>FinalStateParticles_KF</finalStateParticlesColName>
+            <otherElectronsColName>OtherElectrons_KF</otherElectronsColName>
+            <beamPositionX> 0  </beamPositionX>
+            <beamSigmaX> 0.3 </beamSigmaX>
+            <beamPositionY> 0 </beamPositionY>
+            <beamSigmaY> 0.02 </beamSigmaY>
+            <beamPositionZ> 0.0 </beamPositionZ>
+            <trackClusterTimeOffset>25</trackClusterTimeOffset>
```

Comment:
I think this has to be changed to <trackClusterTimeOffset>40</trackClusterTimeOffset> so that the clusters get matched successfully. I tested it with one file but I will run more tests and commit the changes if I am right.

Reply:
Yeah, this is the issue.  I know I had tuned this when I doing my comparisons.  I wonder why it decided to shift...maybe the pulser overlay?  I'll try to look into this ASAP.

Reply:
for data this is 40 which is how I found out